source/adminguide/storage.rst

```diff
@@ -924,6 +927,15 @@ a snapshot request will wait in the queue. If this limit is reached, the
 snapshot request fails and returns an error message.
 
 
+Snapshot Copy
+-------------
```

Comment:
```suggestion
~~~~~~~~~~~~~```

Reply:
Above change is to make the heading font size consistent

![image](https://github.com/apache/cloudstack-documentation/assets/3348673/6b5c7b30-fa18-4289-b4ee-0cea22350729)